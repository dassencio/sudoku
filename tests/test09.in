2 5
 9  _  _  6  _  _  _  _  5  1
 _  _  _  5  _  _  6  _  _  _
 2  _  5  _  _  _  4  _  3  _
 3  6 10  _  _  9  _  _  _  _
 _  8  7  4  _  5  _  _  _  _
 _  _  _  _  9  _  7  6  4  _
 _  _  _  _  8  _  _  5 10  9
 _  2  _  9  _  _  _  8  _  3
 _  _  _  3  _  _ 10  _  _  _
 8  1  _  _  _  _  5  _  _  2
