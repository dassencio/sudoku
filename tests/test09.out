 9  7  8  6 10  2  3  4  5  1
 4  3  2  5  1 10  6  9  8  7
 2  9  5  8  7  1  4 10  3  6
 3  6 10  1  4  9  8  7  2  5
 6  8  7  4  3  5  9  2  1 10
 5 10  1  2  9  3  7  6  4  8
 1  4  3  7  8  6  2  5 10  9
10  2  6  9  5  4  1  8  7  3
 7  5  9  3  2  8 10  1  6  4
 8  1  4 10  6  7  5  3  9  2
